### ISSUE TITLE
Accessibility

### ISSUE THREAD

**Issue by mskelton** [DominicTobias/react-image-crop#314]
First of all, thanks so much for your hard work on this library and making a fantastic image cropper!

Once issue that has come up for us is that this library does not seem to be accessible for users who only use the keyboard.  Do you have any plans on improving the accessibility of this library? 

I found [this library](https://github.com/instructure-react/react-crop) which does have some accessibility, but it does not have even close to the features your library does so switching to that would not be possible for us.  Just thought I would throw it out there for reference.

One other thing I will mention is that I am able to move the crop area when I click on the crop area with my mouse, so it seems like all that this library is missing is tabbing into the crop area, shortcuts for scaling, and shortcuts for expanding horizontally and vertically.

Thanks!

cc: @avo

**Comment by DominicTobias** [comment]
Hello & thanks, seems like it would be a good feature although at the moment I'm working on something else and preparing to move to the other side of the world so I don't have inclination to do the work on it for now. You could:

a) Take a look at `onComponentKeyDown` - https://github.com/DominicTobias/react-image-crop/blob/master/lib/ReactCrop.js#L443 and add more shortcuts to it
b) Handle the shortcuts in your app, since the lib is controlled you can change the crop on shortcuts in the app and pass a new crop in

**Comment by mskelton** [comment]
Sorry for leaving this issue hang out for so long, but I finally am getting some bandwidth to circle back to this.  After talking with a UX designer on my team, there are two things we would like to propose updating to improve the keyboard accessibility experience when using React Image Crop.

First would be to remove the `tabIndex` from the crop selection div in favor of the `tabIndex` on the root div.  Having the tab index on the crop selection results in double tabbing which isn't great.  My previous PR I added is where the second tab index was introduced and I think I just didn't realize there was the other tab index.  I think I must have missed it since the styles set `outline: none` on the cropper div.  While setting the outline to `none` is not good from an accessibility standpoint, this likely is something individual apps need to decide how to handle unless you want to try and provide a solution that works out of the box.

The second improvement would be increasing the nudge sizes.  Currently the small nudge is .2 pixels and the large nudge is 2 pixels.  My recommendation would be to change this to 1 pixel for the small nudge and 10 pixels for the large nudge.  This would match better with other crop experiences.

The final improvement would be to add another shortcut for nudging the crop area.  The shortcut would be cmd/ctrl plus arrow keys for nudging by 100 pixels.  This would compliment the existing nudging shortcuts.

Thoughts?  If you are okay with these changes, I'll open a pull request to implement it.  Thanks again for making this awesome library!

**Comment by DominicTobias-b1** [comment]
No prob your suggestions make a lot of sense to me, if the tabIndex is moved there then `onKeyDown` should move there too which in hindsight makes more sense
